include/odp/api/spec/packet_io.h

```diff
@@ -954,8 +954,8 @@ uint32_t ODP_DEPRECATE(odp_pktio_mtu)(odp_pktio_t pktio);
 /**
  * Enable/Disable promiscuous mode on a packet IO interface.
  *
- * @param[in] pktio	Packet IO handle.
- * @param[in] enable	1 to enable, 0 to disable.
+ * @param pktio   Packet IO handle.
+ * @param enable  1 to enable, 0 to disable.
  *
```

Comment:
This seems a worthwhile cleanup but seems orthogonal to the rest of this PR.

Reply:
This changes the same API header file. Doing it as part of the patch set avoids potential merge conflicts.